moveit_ros/moveit_servo/src/servo.cpp

```diff
@@ -53,13 +53,9 @@ Servo::Servo(ros::NodeHandle& nh, const planning_scene_monitor::PlanningSceneMon
   if (!readParameters())
     exit(EXIT_FAILURE);
 
-  joint_state_subscriber_ = std::make_shared<JointStateSubscriber>(nh_, parameters_.joint_topic);
+  servo_calcs_ = std::make_unique<ServoCalcs>(nh_, parameters_, planning_scene_monitor_, parameter_ns_);
```

Comment:
At this point we should probably check if the state monitor is actually running (check `PSM::getCurrentStateMonitor()`), otherwise call `PSM::startStateMonitor()`. Also, we are using velocity values, so we should call `CSM::enableCopyDynamics()`

Reply:
Good catch, I also needed to wait for the robot state to be complete (i was reading from uninitialized data in one of the tests (casing a bunch of nan's to show up in the output and that was causing the test to fail).  It should all be fixed now.

Reply:
Cheers to unit tests catching something